node/core/dispute-coordinator/src/initialized.rs

```diff
@@ -826,9 +826,29 @@ impl Initialized {
 		let new_state = import_result.new_state();
 
 		let is_included = self.scraper.is_candidate_included(&candidate_hash);
+		let is_backed = self.scraper.is_candidate_backed(&candidate_hash);
+		let is_confirmed = new_state.is_confirmed();
 
 		let potential_spam = !is_included && !new_state.is_confirmed() && !new_state.has_own_vote();
 
+		// Disputes should fall in one of the categories below, otherwise they will be ignored.
+		// `is_included` lands in prioritised queue
+		// `is_confirmed` | `is_backed` lands in best effort queue
+		// The actual prioritisation happens later. At this point irrelevant disputes are filtered out.
+		if !is_included && !is_confirmed && !is_backed {
```

Comment:
We should still import votes (limited by spam slots), we should just not participate. If we also don't import, then the check for confirmed is rather useless - the dispute will never be confirmed, if we don't import votes. (Ok, there is vote chain scraping, but still)

Reply:
For politeness, is there any facility to convince whoever sent up the vote to send us the dispute?